DQMServices/StreamerIO/plugins/TriggerSelector.cc

```diff
@@ -1,58 +1,24 @@
 #include "TriggerSelector.h"
 
+#include "DataFormats/Common/interface/HLTPathStatus.h"
+#include "DataFormats/Common/interface/TriggerResults.h"
+#include "FWCore/Framework/interface/EventSelector.h"
 #include "FWCore/Framework/interface/TriggerNamesService.h"
-#include "FWCore/Utilities/interface/RegexMatch.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/RegexMatch.h"
 
 #include <iostream>
-#include <boost/regex.hpp>
 #include <algorithm>
-#include <cassert>
+
+#include <boost/regex.hpp>
 
 namespace dqmservices {
 
   // compatibility constructor
 
-  TriggerSelector::TriggerSelector(Strings const& pathspecs, Strings const& names) : useOld_(true) {
+  TriggerSelector::TriggerSelector(Strings const& pathspecs, Strings const& triggernames) : useOld_(true) {
     acceptAll_ = false;
-    eventSelector_.reset(new edm::EventSelector(pathspecs, names));
-  }
-
-  TriggerSelector::TriggerSelector(edm::ParameterSet const& config, Strings const& triggernames, bool old_)
-      : useOld_(old_) {
-    acceptAll_ = false;
-    if (old_) {
-      // old mode forced
-      acceptAll_ = false;
-      eventSelector_.reset(new edm::EventSelector(config, triggernames));
-      return;
-    }
-    if (!config.empty()) {
-      // let's see if non empty TriggerSelector is present
-      try {
-        std::string myPath = trim(config.getParameter<std::string>("TriggerSelector"));
-        if (!myPath.empty()) {
-          init(myPath, triggernames);
-          return;
-        }
-      } catch (...) {
-      }
```

Comment:
A string configuration-parameter named `TriggerSelector` is not used anywhere in CMSSW, as far as I can see. Not sure if I need to worry about client code outside CMSSW in this case.